### PR TITLE
Adding val-1fa and val-1fb verification cases

### PR DESCRIPTION
**Pull request by singhgp4321**
Ref: #12

### CODE REVIEW

**Comment by singhgp4321** [comment]
@cticenhour It's still failing the documentation test although I made those updates in the yml file.

**Comment by cticenhour** [comment]
I will take a closer look! Thanks for the heads up.

File: doc/config.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Content:
     - ${ROOT_DIR}/doc/content
     - ${MOOSE_DIR}/framework/doc/content
+    - ${MOOSE_DIR}/modules/heat_conduction/doc/content
```
Comment (cticenhour) [inline doc/config.yml:4]:
```suggestion
    - ${MOOSE_DIR}/modules/heat_conduction/doc/content
    - ${MOOSE_DIR}/modules/ray_tracing/doc/content      # pulled in by heat_conduction
```

You'll need this for the main documentation build (heat conduction has ray tracing as a dependency). I would also explicitly list `RAY_TRACING := yes` in the Makefile for the reminder to the developers that this dependency exists.

File: doc/config.yml [hunk 2]
```diff
@@ -31,6 +32,7 @@ Extensions:
         categories:
             framework: !include ${MOOSE_DIR}/framework/doc/sqa_framework.yml
             tmap: !include ${ROOT_DIR}/doc/sqa_tmap.yml
+            heat_conduction: !include ${MOOSE_DIR}/modules/heat_conduction/doc/sqa_heat_conduction.yml
```
Comment (cticenhour) [inline doc/config.yml:35]:
```suggestion
            heat_conduction: !include ${MOOSE_DIR}/modules/heat_conduction/doc/sqa_heat_conduction.yml
            ray_tracing: !include ${MOOSE_DIR}/modules/ray_tracing/doc/sqa_ray_tracing.yml
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
I can't make these suggestions in the general GitHub interface for some reason, but I would also make these changes: 
```
diff --git a/doc/config.yml b/doc/config.yml
index 4832f50..59fb4ac 100644
--- a/doc/config.yml
+++ b/doc/config.yml
@@ -16,7 +16,7 @@ Extensions:
             Documentation:
               TMAP8-only Syntax: syntax/tmap_only.md
               Complete Code Syntax: syntax/index.md
-            Software Quality: sqa/index.md
+            Software Quality: /sqa/index.md
     MooseDocs.extensions.appsyntax:
         executable: ${ROOT_DIR}
         includes:
diff --git a/doc/sqa_reports.yml b/doc/sqa_reports.yml
index cdcf159..bb89603 100644
--- a/doc/sqa_reports.yml
+++ b/doc/sqa_reports.yml
@@ -24,5 +24,6 @@ Requirements:
         working_dirs:
             - ${ROOT_DIR}/doc/content
             - ${MOOSE_DIR}/framework/doc/content
+            - ${MOOSE_DIR}/modules/heat_conduction/doc/content
         directories:
             - ${ROOT_DIR}/test
```

**Comment by cticenhour** [comment]
All of these changes should fix both the SQA fail and the general documentation build fail.

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
@singhgp4321 Just checking - are you still pulling the verification documentation text from the older TMAP manual? 

It seems that branch protection rules are preventing me from merging this right now, even though I have write access. Pinging @lindsayad about it, though it might be a bit before he is in a position to respond.

**Comment by singhgp4321** [comment]
@cticenhour For this MR I wrote the documentation text, although I try to use what is already in the V&V report.

**Pull request merged**